src/components/Pill/PillButton/PillButton.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import { IoIosArrowUp, IoIosArrowDown } from 'react-icons/io';
+import { CROSS_CHAR, ENTER_KEY } from '../../../constants';
+import styles from './PillButton.scss';
+
+const { block, elem } = bem({
+    name: 'PillButton',
+    classnames: styles,
+    propsToMods: ['isActive', 'isOpen'],
+});
+
+const PillButton = React.forwardRef((props, ref) => {
+    const { isOpen, togglePopup, onClear, name, label, ...rest } = props;
+    const isActive = !!label;
+    const propsForBem = { ...props, isActive };
+
+    let buttonIcon;
+    let useButtonClick = false;
+
+    if (isOpen) {
+        buttonIcon = <IoIosArrowUp />;
+    } else if (isActive) {
+        buttonIcon = CROSS_CHAR;
+        useButtonClick = true;
+    } else {
+        buttonIcon = <IoIosArrowDown />;
+    }
+
+    const buttonClick = useButtonClick
+        ? e => {
+              e.stopPropagation();
+              onClear();
+          }
+        : undefined;
+
+    const handleKeyDownOnButton = useButtonClick
+        ? e => {
```

Comment:
PillButton has an issue with focused clear button. When button focused and push enter in some cases in the console are errors in other cases toggle popup ection is fired two times.